.github/workflows/data-loader-ci.yaml

```diff
@@ -0,0 +1,45 @@
+name: CI for CLI
+
+on:
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  data_loader_ci:
+    name: Build Data Loader CLI
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set version
+        id: version
+        run: |
+          VERSION=$(echo "${GITHUB_REF}" | sed -e "s#refs/tags/v##g")
+          echo "version=${VERSION}" >> $GITHUB_OUTPUT
+
+      - name: Set up JDK 8 and 11 (default 8)
+        uses: actions/setup-java@v4
+        with:
+          distribution: 'temurin'
+          java-version: |
+            11
+            8
+
+      - name: Cache Gradle packages
+        uses: actions/cache@v4
+        with:
+          path: |
+            ~/.gradle/caches
+            ~/.gradle/wrapper
+          key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle*', '**/gradle-wrapper.properties') }}
+          restore-keys: |
+            ${{ runner.os }}-gradle-
+
+      - name: Gradle Data Loader CLI build
+        run: ./gradlew :data-loader:cli:build --no-daemon --stacktrace
+
+      - name: Cleanup Gradle cache
+        run: |
+          rm -f ~/.gradle/caches/modules-2/modules-2.lock
+          rm -f ~/.gradle/caches/modules-2/gc.properties
```

Comment:
If you follow my suggestion of using `gradle/actions/setup-gradle@v4`, this is not necessary.
```suggestion
```

Reply:
Updated